internal/controller/certificates/policies/checks_test.go

```diff
@@ -213,6 +213,31 @@ func Test_NewTriggerPolicyChain(t *testing.T) {
 			message: "Issuing certificate as Secret was previously issued by IssuerKind.new.example.com/testissuer",
 			reissue: true,
 		},
+		"trigger issuance as current CertificateRequest is not signed with private key": {
```

Comment:
Is this title correct? It seems to be testing the case where there is a mismatch between the private key that signed the   CertificateRequest and the private key in Secret.

I suggest 
> trigger if the Secret contains a different private key than was used to sign the CSR.